src/views/Dashboard.tsx

```diff
@@ -191,18 +194,38 @@ const Dashboard = () => {
 
   useKeyPress(["i"], () => setShowAddModal(true), null, true);
 
+  const closeDialog = () => {
+    setShowAddModal(false);
+  }
+
+  React.useEffect(() => {
+    if (showAddModal) {
+      const newTaskModal = modal.confirm({});
+      newTaskModal.update({
```

Comment:
I feel like this should probably be abstracted out of the actual Dashboard function